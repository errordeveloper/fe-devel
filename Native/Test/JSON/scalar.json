6.4839
3.14159265
-4.567
-0.0
0.0
4.5e-2
7.99E30
1e9
-4E-3
