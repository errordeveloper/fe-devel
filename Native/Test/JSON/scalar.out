SCALAR 6.4839
SCALAR 3.14159265
SCALAR -4.567
SCALAR -0
SCALAR 0
SCALAR 0.045
SCALAR 7.99e+30
SCALAR 1000000000
SCALAR -0.004
